x-pack/plugins/lens/public/index.scss

```diff
@@ -0,0 +1,2 @@
+@import 'src/legacy/ui/public/styles/_styling_constants';
+@import './drag_drop/drag_drop';
```

Comment:
There's a file import for './drag_drop/drag_drop' but I don't see the actual file.

Reply:
It's already in the codebase (in a previous PR).